### PR TITLE
Updated Thanos Contributor office Hours in Community.md

### PR DESCRIPTION
**Pull request by DARK-art108**
Signed-off-by: Ritesh Yadav <daydreamingguy941@gmail.com>

## Changes

Added info about Thanos Contributor Office Hours to Thanos Website. Issue #3534 
 - [x] Updated the community.md as requested in PR #3544 
 - [x] Updated Thanos-Office-Contributing-Hour section in Community.md

### CODE REVIEW

**Comment by DARK-art108** [comment]
Hello, @bwplotka I have done the requested changes in this new PR, Please review it (:

Previous PR #3554
Issue #3534

File: docs/contributing/community.md [hunk 1]
```diff
@@ -27,6 +27,10 @@ Everyone is welcome to join & add items to the agenda. The meeting happens in Zo
 
 If you have any suggestions, how we can run the meeting better, please let `@povilasv` or `@metalmatze` know in the CNCF slack.
 
+### Thanos Contributor Office Hours
+
+Thanos Contributor Office Hours are public weekly meetings focused on Thanos development and contributions. It’s meant for developers and maintainers to meet and get unblocked, pair review, and discuss development aspects of the Thanos and related projects. Read more [here](https://docs.google.com/document/d/137XnxfOT2p1NcNUq6NWZjwmtlSdA6Wyti86Pd6cyQhs/edit#heading=h.oy8b2jtupl8m)
```
Comment (soniasingla) [inline docs/contributing/community.md:32]:
```suggestion
Thanos Contributor Office Hours are public weekly meetings focused on Thanos development and contributions. It’s meant for developers and maintainers to meet, get unblocked, pair review, and discuss development aspects of Thanos and related projects. Read more [here](https://docs.google.com/document/d/137XnxfOT2p1NcNUq6NWZjwmtlSdA6Wyti86Pd6cyQhs/edit#heading=h.oy8b2jtupl8m).
```

**Review by soniasingla — CHANGES_REQUESTED** [review]

**Comment by soniasingla** [comment]
@DARK-art108 thanks, it looks good, only one nit change (:

**Comment by DARK-art108** [comment]
Ok sure @soniasingla I will do it. :)

File: docs/contributing/community.md (hunk 1 above)
Comment (bwplotka) [reply docs/contributing/community.md:32]: 👍🏽

**Review by bwplotka — APPROVED** [review]
Amazing. Thank you! LGTM once @soniasingla comments are addressed 🤗

**Comment by DARK-art108** [comment]
@soniasingla Please review the changes :)

**Review by soniasingla — APPROVED** [review]
Thanks @DARK-art108 for your contribution 🎉

**Pull request merged**